bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -9,8 +9,8 @@ extendedArguments:
   - "/etc/kubernetes/secrets/kube-ca.crt"
   cluster-signing-key-file:
   - "/etc/kubernetes/secrets/kube-ca.key"
-  port:
-  - "0"
```

Comment:
@deads2k @sttts this is new, we were not doing liveness probe during bootstrap, I wonder if we want to start now.